packages/dashboard-core-plugins/src/panels/MarkdownNotebook.tsx

```diff
@@ -16,8 +18,9 @@ import {
   ReactMarkdownProps,
   TransformImage,
   TransformLink,
-} from 'react-markdown/src/ast-to-react';
+} from 'react-markdown/lib/ast-to-react';
```

Comment:
I know this isn't code you added, but you should be able to remove the `/lib/ast-to-react` imports by adding `import { Components, Options } from 'react-markdown'` and then one of those should have these types available as either `A.TransformImage` or `A['TransformImage']` I believe. Give it a shot and let me know if you run into any issues with it